Wox/SettingWindow.xaml.cs

```diff
@@ -66,10 +66,21 @@ private void Setting_Loaded(object sender, RoutedEventArgs ev)
                 UserSettingStorage.Instance.Save();
             };
 
+            comboMaxResultsToShow.SelectionChanged += (o, e) =>
+            {
+                UserSettingStorage.Instance.MaxResultsToShow = (int)comboMaxResultsToShow.SelectedItem;
+                UserSettingStorage.Instance.Save();
+                MainWindow.pnlResult.lbResults.GetBindingExpression(MaxHeightProperty).UpdateTarget();
```

Comment:
Since there is no proper databinding in place, I had to refresh the height this way :/